src/routes/v1/apps/handlers/createApp.js

```diff
@@ -223,14 +223,20 @@ module.exports = {
                     'Inserting logo metadata to db and link it to the appVersion'
                 )
                 const imageFileMetadata = imageFile.hapi
-
+                const [imageInfo] = appJsonPayload.images
+                let caption, description
+                if (imageInfo) {
+                    ;({ caption, description } = imageInfo)
+                }
                 const { id, uuid } = await addAppVersionMedia(
                     {
                         userId: requestUserId,
                         appVersionId: appVersion.id,
                         imageType: ImageType.Logo,
                         fileName: imageFileMetadata.filename,
                         mime: imageFileMetadata.headers['content-type'],
+                        caption: caption,
+                        description: description,
```

Comment:
Note that these can be undefined. Shorthand would work, but would give undefined: undefined, which is not prefereable.